tests/webview/ui/test_content.py

```diff
@@ -766,3 +765,38 @@ def test_chapter_review_version_matches_book_version(webview_base_url, selenium,
 
     # THEN the chapter review version matches the book version
     assert content.page_version == content.book_version
+
+
+@markers.webview
+@markers.test_case('C195064')
+@markers.nondestructive
+@markers.parametrize('ch_review_id', ['4fGVMb7P@1'])
+def test_go_to_first_page(webview_base_url, selenium, ch_review_id):
```

Comment:
this test name needs to be improved. What is `test_go_to_first_page`? If someone was looking at this it would be hard to tell what the intent of the test was by the name.

a suggestion:

`test_books_containing_go_to_book_link`